front/src/modules/ui/modal/components/ConfirmationModal.tsx

```diff
@@ -13,7 +13,7 @@ import {
 } from '@/ui/section/components/Section';
 import { H1Title, H1TitleFontColor } from '@/ui/typography/components/H1Title';
 
-interface ConfirmationModalProps {
```

Comment:
We prefer to use type and be consistent with it

Reply:
yes!